arbos/arbosState/arbosstate.go

```diff
@@ -204,7 +204,7 @@ func InitializeArbosState(stateDB vm.StateDB, burner burn.Burner, chainConfig *p
 
 	// Solidity requires call targets have code, but precompiles don't.
 	// To work around this, we give precompiles fake code.
-	for _, genesisPrecompile := range getArbitrumOnlyGenesisPrecompiles(chainConfig) {
+	for _, genesisPrecompile := range GetArbitrumOnlyGenesisPrecompiles(chainConfig) {
 		stateDB.SetCode(genesisPrecompile, []byte{byte(vm.INVALID)})
```

Comment:
Also, we shouldn't be setting code for the secp256r1 precompile. It's not really an "arbitrum only" precompile.

Reply:
I don't think we want to set the code since calls to it won't be ABI encoded and will be raw calls